src/enki_msg.py

```diff
@@ -52,4 +52,4 @@ def print_report(self, start_time: float, output: str = None) -> None:
 
         for category, files in self.report.items():
             logging.error(f"{category} found in the following files:")
-            logging.error('\t' + separator.join(files))
+            print('\t' + separator.join(files))
```

Comment:
Careful here. There's a semantic and functional difference between logging and printing:

* By default, logging uses stderr and printing uses stdout. That matters if you pipe the output of enki. The heading of the message and the list of files could end up in different places.

* Printing always displays the message. Logging depends on the configuration at the start of the program, which controls the log level, or even if the logs should display at all. As a result, the list of files might appear without the heading.

Reply:
Ah, I see. Thanks for the tip. I addressed it in this commit [d48c86f](https://github.com/Levi-Leah/enki/pull/40/commits/d48c86fc9561423285cb6d3006698945cdcb586f)